00-master.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+set -o nounset -o errexit
+
+kubeadm init --pod-network-cidr=10.244.0.0/16 --apiserver-advertise-address=${MASTER_PRIVATE_IP} --apiserver-cert-extra-sans=${MASTER_PUBLIC_IP}
```

Comment:
Maybe it's a good idea to consider adding hostname to `apiserver-cert-extra-sans`.

Reply:
Seems reable to me, @xmudrii , but we can merge without and add that afterwards.